integration/tests/cook/test_basic.py

```diff
@@ -2068,25 +2068,7 @@ def test_default_container_for_pool(self):
             max_retries=5)
         self.assertEqual(resp.status_code, 201)
         job = util.wait_for_job(self.cook_url, job_uuid, 'completed')
-        default_image_config = expected_container["docker"]["image"]
```

Comment:
Can this test be repaired in a useful way instead of basically deleted?

Reply:
I tried to think of a way but it was too close to re-implementing the business logic in a test for me. I'd rather have it like this.